src/DynamoCoreWpf/Views/Search/LibrarySearchView.xaml.cs

```diff
@@ -31,8 +32,19 @@ public LibrarySearchView()
 
         private void OnLibrarySearchViewLoaded(object sender, RoutedEventArgs e)
         {
-            (DataContext as SearchViewModel).SearchTextChanged +=
-                (s, eInner) => topResultPanel.BringIntoView();
+            viewModel = DataContext as SearchViewModel;
+            viewModel.SearchTextChanged += OnSearchTextBoxKeyDown;
+        }
+
+        private void OnSearchTextBoxKeyDown(object sender, EventArgs e)
```

Comment:
A documentation for this method would be nice:

```csharp
// Changing text content of the search box should always bring up the the 
// "top result" (since there is no search term). Also, any possible tool-tip
// that is displayed and highlighted item should be dismissed right away.
```